src/sentry/static/sentry/app/views/projectDetail/projectScoreCards/projectApdexScoreCard.tsx

```diff
@@ -72,6 +82,38 @@ class ProjectApdexScoreCard extends AsyncComponent<Props, State> {
     return endpoints;
   }
 
+  /**
+   * If there's no apdex in the time frame, check if there is one in the last 90 days (empty message differs then)
+   */
+  async onLoadAllEndpointsSuccess() {
+    const {organization, selection} = this.props;
+    const {projects} = selection;
+
+    if (defined(this.currentApdex) || defined(this.previousApdex)) {
+      this.setState({noApdexEver: false});
+      return;
+    }
+
+    this.setState({loading: true});
+
+    const response = await this.api.requestPromise(
```

Comment:
should we add a `try...catch` here?

Reply:
I do not want to handle that, at least for now (still playing with it).